src/sentry/snuba/metrics/extraction.py

```diff
@@ -998,6 +999,11 @@ def field_to_extract(self):
 
         return self._arguments[0]
 
+    @property
+    def metric_type(self) -> str:
+        """Returns c, d or s representing if it's a counter, distribution or set."""
```

Comment:
This is mostly useful within tests so I could get a docstring when hovering on the IDE.